tests/console/force_scheduled_tasks.pm

```diff
@@ -47,7 +47,7 @@ sub run {
     record_soft_failure 'bsc#1063638 - review I/O scheduling parameters of btrfsmaintenance' if (time - $before) > 60 && get_var('SOFTFAIL_BSC1063638');
     # Disable cron jobs on older SLE12 by symlinking them to /bin/true
     if (!get_var('SOFTFAIL_BSC1063638') && script_run("! [ -d /usr/share/btrfsmaintenance/ ]")) {
-        assert_script_run('find /usr/share/btrfsmaintenance/ -type f -exec ln -fs /bin/true {} \;');
+        assert_script_run('find /usr/share/btrfsmaintenance/ -type f -exec ln -fs /bin/true {} \;', timeout => 310);
```

Comment:
in ticket you saying that @foursixnine suggested 2 minutes but you putting 5. And we speaking here about querying folder and creating symlinks for found items . Do we really have system where it taking even 2 minutes ????

Reply:
He suggested increasing by 2 Minutes, the default timeout is 90 seconds adding 120 seconds makes 210 seconds, which means I failed at calculating.
Apaprently we need it, otherwise we would not have the failure in the ticket?
Will adapt this tomorrow ^^

Reply:
@asmorodskyi that's the cool thing: Even that simple operation is stalled in our tests what you actually just *can* describe as bsc#1063638 :)